docs/faq.md

```diff
@@ -28,6 +40,17 @@ To change your password, sign in to the [Developers portal](https://developers.a
 
 If you haven't received a confirmation mail, it is often because the email address was entered incorrectly. Please sign in to the [Developers portal](https://developers.amadeus.com/){:target="\_blank"} and visit the **My Account** section to confirm that the email address used to create the account is correct. If so, please check your spam folder for an email from noreply@amadeus.com.
 
+## Business enquiries
+
+### How can I monetise my application?
+
+You are free to create your own business models around our APIs, such as charging users to use our APIs in their apps or adopting a subscription-based model. However, we do not give any commission or other types of incentives for the Self-Service API. The latter is only possible in the Enterprise framework.
```

Comment:
In the case of flight booking, users can also monetise their apps by adding a markup on flight offers.